src/libusermode/userhook.cpp

```diff
@@ -848,7 +860,7 @@ std::optional<HookActions> get_hook_actions(const std::string& str)
     return std::nullopt;
 }
 
-void drakvuf_load_dll_hook_config(drakvuf_t drakvuf, const char* dll_hooks_list_path, const bool print_no_addr, std::vector<plugin_target_config_entry_t>* wanted_hooks)
+void drakvuf_load_dll_hook_config(drakvuf_t drakvuf, const char* dll_hooks_list_path, const bool print_no_addr, hook_filter_t const& hook_filter, wanted_hooks_t* wanted_hooks)
```

Comment:
Why not `wanted_hooks_t&` ? Would make code below clearer.

Reply:
Done